_notebooks/2020-03-27-covid19-progress-projections.ipynb

```diff
@@ -0,0 +1,2126 @@
+{
```

Comment:
<ul><li>"New Deaths in 5 days" needs to be "New Deaths in last 5 days"</li><li>I think you can add ICU capacity into this chart. I Googled "ICU capacity by country" and<a href="https://en.wikipedia.org/wiki/List_of_countries_by_hospital_beds" target="_blank"> this wikipedia article</a> was the third result, which has a table of data. Instead of ICU capacity per 100k, suggest showing "Estimated percent of ICU capacity..."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/github/covid19-dashboard/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>